tests/cross/src/test/scala/tests/pc/InlayHintsSuite.scala

```diff
@@ -16,6 +16,302 @@ class InlayHintsSuite extends BaseInlayHintsSuite {
     }
   }
 
+  check(
+    "single-chain-same-line",
+    """object Main{
+      |  trait Bar {
+      |   def bar: Bar
+      |  }
+      |
+      |  trait Foo {
+      |    def foo(): Foo
+      |  }
+      |
+      |val bar: Bar = ???
+      |val foo: Foo = ???
+      |
+      |val thing1: Bar = bar.bar
+      |val thing2: Foo = foo.foo()
+      |}
+      |""".stripMargin,
+    """object Main{
+      |  trait Bar {
+      |   def bar: Bar
+      |  }
+      |
+      |  trait Foo {
+      |    def foo(): Foo
+      |  }
+      |
+      |val bar: Bar = ???
+      |val foo: Foo = ???
+      |
+      |val thing1: Bar = bar.bar
+      |val thing2: Foo = foo.foo()
+      |}
+      |""".stripMargin
+  )
+
+  check(
+    "multi-chain-same-line",
+    """object Main{
+      |  trait Bar {
+      |   def bar: Bar
+      |  }
+      |
+      |  trait Foo {
+      |    def foo(): Foo
+      |  }
+      |
+      |val bar: Bar = ???
+      |val foo: Foo = ???
+      |
+      |val thing1: Bar = bar.bar.bar
+      |val thing2: Foo = foo.foo().foo()
+      |}
+      |""".stripMargin,
+    """object Main{
+      |  trait Bar {
+      |   def bar: Bar
+      |  }
+      |
+      |  trait Foo {
+      |    def foo(): Foo
+      |  }
+      |
+      |val bar: Bar = ???
+      |val foo: Foo = ???
+      |
+      |val thing1: Bar = bar.bar.bar
+      |val thing2: Foo = foo.foo().foo()
+      |}
+      |""".stripMargin
+  )
+
+  check(
+    "single-chain-new-line",
+    """object Main{
+      |  trait Bar {
+      |   def bar: Bar
+      |  }
+      |
+      |  trait Foo {
+      |    def foo(): Foo
+      |  }
+      |
+      |val bar: Bar = ???
+      |val foo: Foo = ???
+      |
+      |val thing1: Bar = bar
+      |  .bar
+      |val thing2: Foo = foo
+      |  .foo()
+      |}
+      |""".stripMargin,
+    """object Main{
+      |  trait Bar {
+      |   def bar: Bar
+      |  }
+      |
+      |  trait Foo {
+      |    def foo(): Foo
+      |  }
+      |
+      |val bar: Bar = ???
+      |val foo: Foo = ???
+      |
+      |val thing1: Bar = bar
+      |  .bar/*: Bar<<(2:8)>>*/
```

Comment:
I figured it makes sense to show the types on one-liners if you indent the call. This can easily be removed if people don't like it

Reply:
I decided to not do this anymore. Theres just too many one liners that look bad with it